Makefile

```
@@ -67,7 +67,10 @@ uninstall:
 		"$(DEST)/bin/limactl" \
 		"$(DEST)/bin/nerdctl.lima" \
 		"$(DEST)/share/lima" "$(DEST)/share/doc/lima"
-	# TODO: remove $(DEST)/bin/nerdctl only when it is a symlink to nerdctl.lima
+	@test -L "$(DEST)/bin/nerdctl" && \
+		(if [ "$(shell readlink $(DEST)/bin/nerdctl)" = "$(DEST)/bin/nerdctl.lima" ]; then \
+			unlink "$(DEST)/bin/nerdctl"; \
+		fi)
```

I don't think this code works because the symlink value is just `nerdctl.lima` and does not include the absolute path.

I'm also somewhat bothered by the fact that the `$(shell  readlink ...)` call is executed while the Makefile is being expanded while the check that the file is a symlink only happens during command execution, i.e. afterwards.

But since `readlink` just returns the empty string when the target is not a symlink, we don't really need that test at all, and the expression could be simply:

```suggestion
	if [ "$$(readlink "$(DEST)/bin/nerdctl")" = "nerdctl.lima" ]; then rm "$(DEST)/bin/nerdctl"; fi
```

---

Thanks for the additional comment. Fixed.